data/RTTR/CAMPAIGNS/roman/campaign.lua

```diff
@@ -0,0 +1,29 @@
+rttr:RegisterTranslations(
```

Comment:
do we have something like `getRequiredLuaVersion` for campaign luas too? that would be good to add something like that from the start, so we can add newer features later easily

Reply:
Introduced